src/views/pos-test.fxml

```diff
@@ -1,29 +1,28 @@
-      <Text layoutX="845.0" layoutY="180.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Individual Items" wrappingWidth="184.21875" />
-      <TableView fx:id="receiptTableView" layoutX="39.0" layoutY="164.0" prefHeight="359.0" prefWidth="387.0" />
-      <Text layoutX="39.0" layoutY="141.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Receipt Table" wrappingWidth="97.21875" />
-      <Text layoutX="258.0" layoutY="604.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Total:" wrappingWidth="69.0908203125">
-         <font>
-            <Font size="18.0" />
-         </font>
-      </Text>
-      <TextField fx:id="totalField" layoutX="323.0" layoutY="584.0" prefHeight="27.0" prefWidth="97.0" />
-      <Text layoutX="258.0" layoutY="564.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Tax:" wrappingWidth="69.0908203125">
-         <font>
-            <Font size="18.0" />
-         </font>
-      </Text>
-      <TextField fx:id="taxField" layoutX="323.0" layoutY="544.0" prefHeight="27.0" prefWidth="97.0" />
-   </children>
-</Pane>
+<!--<Pane fx:id="backPane" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="799.0" prefWidth="1279.0" stylesheets="@../stylesheets/posStyles.css" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="controllers.Controller">-->
+   <!--<children>-->
+      <!--<ListView fx:id="categoryListView" layoutX="526.0" layoutY="232.0" prefHeight="335.0" prefWidth="227.0" />-->
+      <!--<ListView fx:id="foodItemsListView" layoutX="845.0" layoutY="193.0" prefHeight="560.0" prefWidth="338.0" />-->
+      <!--<Text layoutX="526.0" layoutY="222.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Categories" wrappingWidth="97.21875" />-->
+      <!--<Text layoutX="845.0" layoutY="180.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Individual Items" wrappingWidth="184.21875" />-->
+      <!--<TableView fx:id="receiptTableView" layoutX="39.0" layoutY="164.0" prefHeight="359.0" prefWidth="387.0" />-->
+      <!--<Text layoutX="39.0" layoutY="141.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Receipt Table" wrappingWidth="97.21875" />-->
+      <!--<Text layoutX="258.0" layoutY="604.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Total:" wrappingWidth="69.0908203125">-->
+         <!--<font>-->
+            <!--<Font size="18.0" />-->
+         <!--</font>-->
+      <!--</Text>-->
+      <!--<TextField fx:id="totalField" layoutX="323.0" layoutY="584.0" prefHeight="27.0" prefWidth="97.0" />-->
+      <!--<Text layoutX="258.0" layoutY="564.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Tax:" wrappingWidth="69.0908203125">-->
+         <!--<font>-->
+            <!--<Font size="18.0" />-->
+         <!--</font>-->
+      <!--</Text>-->
+      <!--<TextField fx:id="taxField" layoutX="323.0" layoutY="544.0" prefHeight="27.0" prefWidth="97.0" />-->
+   <!--</children>-->
+<!--</Pane>-->
```

Comment:
I can't really tell from the diff, but is this whole file commented out?  if so, then just throw away the file.  you don't need it.